NAudio/Wave/WaveInputs/WasapiCaptureRT.cs

```diff
@@ -148,6 +148,14 @@ private void InitializeCaptureDevice(IAudioClient audioClientInterface)
             if (waveFormat == null)
             {
                 waveFormat = audioClient.MixFormat;
+                if (mix.BitsPerSample == 32)
+                {
+                    waveFormat = WaveFormat.CreateIeeeFloatWaveFormat(mix.SampleRate, mix.Channels);
+                }
+                else
+                {
+                    waveFormat = new WaveFormat(mix.SampleRate, mix.BitsPerSample, mix.Channels);
+                }                
```

Comment:
I suggest simplifying:

```suggestion
                waveFormat = mix.BitsPerSample == 32
                    ? WaveFormat.CreateIeeeFloatWaveFormat(mix.SampleRate, mix.Channels)
                    : new WaveFormat(mix.SampleRate, mix.BitsPerSample, mix.Channels);           
```